guides/advanced_index_actions.md

```diff
@@ -6,10 +6,11 @@ In this guide, we will look at some advanced index actions that are not covered
 Let's create a client instance, and an index named `movies`:
 ```javascript
 const { Client } = require('@opensearch-project/opensearch');
+
 const client = new Client({
```

Comment:
why not point to createSecureClient() here?

Reply:
Beecause that's a helper function for our test suites only. The function creates a client using `admin:admin` or `admin:myStrongPassword123!` depending on the OPENSEARCH_VERSION env. Users should not use that function because their password will be different.

Reply:
ahh I see. ty for explanation!!